src/component-material.tsx

```diff
@@ -103,7 +103,7 @@ export const ComponentMaterial = React.forwardRef(function ComponentMaterial(
         }
 
         return acc
-      }, DEFAULT_STATE),
+      }, Object.assign({}, DEFAULT_STATE)),
```

Comment:
Should we use the more modern syntax with `{ ...DEFAULT_STATE }` ?

Reply:
sure, i'll update it